core/src/main/java/jenkins/monitor/JavaVersionRecommendationAdminMonitor.java

```diff
@@ -46,9 +47,11 @@
 @Symbol("javaVersionRecommendation")
 public class JavaVersionRecommendationAdminMonitor extends AdministrativeMonitor {
 
+    private static Boolean disabled = SystemProperties.getBoolean(JavaVersionRecommendationAdminMonitor.class.getName() + ".disabled", false);
```

Comment:
PR to jenkins.io to add to docs?

Reply:
Sure: https://github.com/jenkins-infra/jenkins.io/pull/4469